liquid_node/backup.py

```diff
@@ -223,10 +237,10 @@ def is_index_available(es_client, name):
 
 
 @retry()
-def backup_collection_es(dest, name):
+def backup_collection_es(dest, name, es_url):
     tmp_file = dest / "es.tgz.tmp"
     log.info(f"Dumping collection {name} es snapshot to {tmp_file}")
-    es = JsonApi(f"http://{nomad.get_address()}:9990/_es")
+    es = JsonApi(f"http://{nomad.get_address()}:9990{es_url}")
```

Comment:
Best call this argument `es_url_suffix` since it's just the part after the port, not the whole URL.